pkg/image/common/validator.go

```diff
@@ -186,6 +202,20 @@ func (v *vmiValidator) CheckSecurityParameters(vmi *v1beta1.VirtualMachineImage)
 	return nil
 }
 
+// CheckPVCInUse checks if the PVC is in use by any pods, this is only used to CDI backend
```

Comment:
> this is only used to CDI backend

this looks like a general function that can be re-used. if not, then consider reinforcing it in the code, with something like `if vmi.Spec.Backend != VMIBackendCDI { return nil }` (or `return err`)?

Reply:
Nice suggestion!

Updated!